lib/xlsx/xform/sheet/sheet-view-xform.js

```diff
@@ -43,6 +43,7 @@ utils.inherits(SheetViewXform, BaseXform, {
         xmlStream.addAttribute(name, value);
       }
     };
+    add('rightToLeft', '1', model.rtl === true);
```

Comment:
there needs to be a corresponding line in the parseOpen function or it won't be able to read files properly